test.py

```diff
@@ -1,8 +1,34 @@
 import pandas as pd 
```

Comment:
so these are the real changes and were I will keep working, not in the notebook 
also better for interacting with the charts (zooming, etc.)